packages/utilities/src/styled.test.tsx

```diff
@@ -63,6 +63,34 @@ describe('styled', () => {
     Stylesheet.getInstance().reset();
   });
 
+  it('can create pure components', () => {
+    let renderCount = 0;
+    const render = () => {
+      renderCount++;
+      return <div />;
+    };
+    const styles = () => ({});
+    const Comp = styled(render, styles);
+    const PureComp = styled(render, styles, undefined, undefined, true);
+    const App = () => {
+      return (
+        <div>
+          <Comp />
+          <PureComp />
+        </div>
+      );
+    };
+    const appWrapper = mount(<App />);
+
+    try {
+      expect(renderCount).toEqual(2);
+      appWrapper.setProps({ 'data-foo': '1' });
+      expect(renderCount).toEqual(3);
```

Comment:
I wonder if it would be better just to make sure that the right component rendered, as opposed to just one of them (ruling out inversions of behavior.)